### PR TITLE
bpython: add page

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: \`common\`, \`linux\`, \`osx\`, \`windows\`, \`sunos\`, \`android\`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/bpython.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# bpython
+
+> A fancy interface to the Python interpreter.
+> Provides syntax highlighting and many other nice-to-haves in REPL mode.
+> More information: <https://docs.bpython-interpreter.org/en/latest/>.
```
Comment (spageektti) [inline pages/common/bpython.md:5]: Is there a better link? This one doesnt show information about how to use the tool

Comment (gutjuri) [reply pages/common/bpython.md:5]: There's also <https://bpython-interpreter.org/> but I don't know if it is more helpful.

Comment (gutjuri) [reply pages/common/bpython.md:5]: Nevermind, I found https://manned.org/man/bpython

Should we take this one even though it is not "official"?

Comment (spageektti) [inline pages/common/bpython.md:5]:
```suggestion
> More information: <https://manned.org/bpython>.
```

Comment (fazlearefin) [reply pages/common/bpython.md]: I would say we should use this https://docs.bpython-interpreter.org/en/latest/ as it is official and will be actively maintained.

**Review by spageektti — APPROVED** [review]

**Pull request merged**